doc/man1/openssl-ca.pod.in

```diff
@@ -69,15 +69,19 @@ B<openssl> B<ca>
 
 =head1 DESCRIPTION
 
-This command is a minimal CA application. It can be used
-to sign certificate requests in a variety of forms and generate
-CRLs. It also maintains a text database of issued certificates
-and their status.
-When signing certificates, a single certificate request can be specified
+This command emulates a CA application but should not be used in production.
```

Comment:
The ship has already sailed. I do not think the ca cannot be used for internal ca purposes. Of course I would not recommend to base a public CA on it although I've seen such in the year 2000 or so.

Reply:
Still I think the above recommendation would be good to follow.
The code of the OpenSSL apps is not in good shape
and having CA private keys in files is not advisable.

Reply:
Then please explain the deficiencies but please do not be paternalizing. As I said, people are using the ca anyway in production for various internal CA purposes. Also cannot be the CA private key actually stored in smart-card or HSM via an engine?

Reply:
All right, done. 
Also added the explicit advice to use secure private key storage when using the `ca` app productively.